### DISCUSSION TITLE
Apollo Save Tool v1.2.2

### DISCUSSION THREAD

**Discussion by bucanero** [bucanero/apollo-ps4#82] [Announcements]
### Changes

* New Save Wizard codes
* Custom decryption support
  * Diablo 3
* Changed background music
* Removed unused screen settings

### Patch Engine

* Updated Apollo patch engine v0.3.0
* Improve patch error handling
* Save Wizard / Game Genie
  * Improve SW code types 9, A
  * Add SW code types 3, 7, B, C, D
* BSD scripts
  * New commands: `copy`, `endian_swap`, `msgbox`
  * New custom hash: `force_crc32`, `mgspw_checksum`
  * Support initial value for `add/wadd/dwadd/wsub`
  * Fix `md5_xor`, `sha1_xor64` custom hashing
  * Fix little-endian support for decrypters/hashes

<hr /><em>This discussion was created from the release <a href='https://github.com/bucanero/apollo-ps4/releases/tag/v1.2.2'>Apollo Save Tool v1.2.2</a>.</em>

**Comment by muzammil12398** [comment]
hello sir i need save file for god of war rafnarok

**Reply by Masamune3210** [reply]
This is not the place to ask for saves, go somewhere else
